### ISSUE TITLE
latex drawer fails with discontiguous multiple controlled gate target

### ISSUE THREAD

**Issue by 1ucian0** [Qiskit/qiskit-terra#5969]
Similar to https://github.com/Qiskit/qiskit-terra/issues/5968, the latex drawer does not support that the target gate is not in contiguous wires:

```python
from qiskit import QuantumCircuit
from qiskit.circuit.library import MCXVChain

circuit = QuantumCircuit(6)
circuit.append(MCXVChain(3), [0, 1, 2, 3, 5], [])
```
![image](https://user-images.githubusercontent.com/766693/110032768-d29bd280-7d38-11eb-8e8f-a47a8861c650.png)

Here is the `text` output for comparison:
```
q_0: ──■───
       │   
q_1: ──■───
       │   
q_2: ──■───
     ┌─┴──┐
q_3: ┤0   ├
     │    │
q_4: ┤  X ├
     │    │
q_5: ┤1   ├
     └────┘
```

**Comment by enavarro51** [comment]
@1ucian0 This is already fixed in #5855 and a revised #5551 could add the wire numbers.

![image](https://user-images.githubusercontent.com/16268251/110056323-6fe41000-7d13-11eb-9e99-6889cb7bae46.png)

**Comment by TharrmashasthaPV** [comment]
@1ucian0 . I think we can close this issue since this issue has been fixed in #5855 and #5551 as Edwin mentioned.

**Comment by 1ucian0** [comment]
Thanks! Closing.
